collections/_release-notes/2025-08-05-1-205-0.md

```diff
@@ -0,0 +1,14 @@
+---
+title: 08.05.2025
+layout: productupdates
+tags:  bugfix sdk 
+date: 2025-08-05
+
+---
+
+Today’s release includes the following updates:
+
+- Support ellipsis use in SDK sandbox environment
+- Change MANIFEST.json default fields (non-breaking)
```

Comment:
Can we specify this is SDK plugin related. Also not sure if we want to but non-breaking in parenthesis, seems weird to put in release notes. The assumption is if it isnt called out as a breaking change it it good. 

Also is the MANIFEST.json documented anywhere where we can link them to in order to read more about what these default values mean?